### ISSUE TITLE
can't find yolov8s-seg.yaml or yolov8n-p2.yaml

### ISSUE THREAD

**Issue by minwim** [ultralytics/ultralytics#2649]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Where can I find the file yolov8s-seg.yaml or yolov8n-p2.ymal,????
I can't find 
but i can use as below:
_model = YOLO('yolov8s-seg.yaml')_
why?

### Additional

Where can I find the file yolov8s-seg.yaml or yolov8n-p2.ymal,????
I can't find 
but i can use as below:
_model = YOLO('yolov8s-seg.yaml')_
why?

**Comment by minwim** [comment]
ok

**Comment by glenn-jocher** [comment]
@minwim, the `yolov8s-seg.yaml` and `yolov8n-p2.yaml` files are not included in this repository. However, you can still use them if you provide them yourself. You can create or download the needed files and put them in the same directory as your code. Then, you can refer to them by filename like `_model = YOLO('yolov8s-seg.yaml')_`. Let me know if you have any other questions or if there's anything else I can help with.

**Comment by ZhaoChunjiang** [comment]
Please tell me the format of these yaml files or where can I download “the needed files” from.

**Comment by glenn-jocher** [comment]
@ZhaoChunjiang hi there! Thanks for your follow-up question. 

The YAML files you mentioned, `yolov8s-seg.yaml` and `yolov8n-p2.yaml`, are configuration files that typically define the architecture and hyperparameters of the model. However, these specific files are not provided in this repository.

As for their format, each YAML configuration file generally includes critical sections like:
- `architecture`: defining the layers and specifics of the network architecture.
- `hyperparameters`: specifying learning rates, weight decay, and other training-related parameters.
- `dataset`: mentioning your training and validation dataset paths, number of classes, etc.

I encourage you to look at the provided YAML files in the repository to grasp the structure. They are designed to be as user-friendly as possible and modifying them for your own needs is usually straightforward. 

Regrettably, the Ultralytics YOLOv8 repository does not provide a direct download or access to the YAML files you mentioned. They might be custom YAML files that were perhaps utilised by another user in their project. If you have seen these filenames in another project, it may be beneficial to check there.

I hope this information is helpful, but don't hesitate to ask if you have any more questions.
